tmt/steps/prepare/repository.py

```diff
@@ -0,0 +1,97 @@
+from typing import Any, Optional
+
+import click
+
+import tmt
+import tmt.steps.prepare
+import tmt.utils
+from tmt.steps.provision import Guest
+
+
+class PrepareRepository(tmt.steps.prepare.PreparePlugin):
+    """
+    Prepare custom repository
+    Example config:
+    prepare:
+        how: repository
+        name: Local repository
+        baseurl: file:///repository
+        gpgcheck: 0
+        gpgkey: file:///gpgkey
+        skip-if-unavailable: 1
+        order: 1
+        priority: 99
+    """
+
+    # Supported methods
+    _methods = [tmt.steps.Method(name='repository', doc=__doc__, order=50)]
```

Comment:
The way how steps register methods has changed, see other plugins.
Hint: `@tmt.steps.provides_method('repository')`